index.php

```diff
@@ -15,18 +15,29 @@
     exit();
 }
 
-$db = false;
-if ($config['db']['connect']) {
-    $db = connectToDB($config['db']);
-}
+$installed_directory = __DIR__;
+$cache_location = $config['cache_location'];
```

Comment:
Should be optional. I would expect a function get_cache_config which returns false if not set, otherwise the defaults for path/timeout, unless customer did define a value